iOS/Core/PixelEvent.swift

```diff
@@ -1784,6 +1786,8 @@ extension Pixel.Event {
         case .networkProtectionActivationRequestFailed: return "m_netp_network_extension_error_activation_request_failed"
         case .networkProtectionDisconnected: return "m_netp_vpn_disconnect"
         case .networkProtectionNoAccessTokenFoundError: return "m_netp_no_access_token_found_error"
+        case .networkProtectionVPNAccessRevoked: return "m_vpn_access_revoked"
```

Comment:
For consistency shouldn't we stick to the `netp` prefix?

Reply:
It's ok to change.  The name "NetworkProtection" is long gone and I'd like to just stick to VPN for anything new.

Old pixels and code will be renamed eventually (or not and have little impact).  But generally speaking I embrace some level of chaos to move forward 😄